src/solvers/dgmulti/types.jl

```diff
@@ -78,6 +78,18 @@ function DGMulti(element_type::AbstractElemShape,
   return DG(rd, nothing #= mortar =#, surface_integral, volume_integral)
 end
 
+DGMulti(basis::RefElemData; volume_integral, surface_integral) =
+  DG(basis, nothing #= mortar =#, surface_integral, volume_integral)
+
+"""
+  DGMultiBasis(element_type, polydeg; approximation_type = Polynomial(), kwargs...)
```

Comment:
```suggestion
    DGMultiBasis(element_type, polydeg; approximation_type = Polynomial(), kwargs...)
```
Needs 4 spaces to be rendered as code